### PR TITLE
Feature add chapter stars update colours

### PR DESCRIPTION
**Pull request by mareliefi**

### CODE REVIEW

File: templates/activity.mustache [hunk 1]
```diff
@@ -64,9 +64,40 @@
     <!-- Only display mastery for practice activities. -->
     {{#sectionid}}
     <span class="emphasis-bold answer">Chapter Mastery (<span id="chapter-mastery-title"></span>):</span><br>
-    <progress class="progress" id="chapter-mastery" value="0" max="100" data-text="0%"></progress><br>
+        <svg style="display:none;">
```
Comment (HAASLEWER) [inline templates/activity.mustache:67]: Should this be hidden?

File: templates/activity.mustache [hunk 2]
```diff
@@ -64,9 +64,40 @@
...
+            <defs>
+                <symbol id="fivestars">
```
Comment (HAASLEWER) [inline templates/activity.mustache:69]: Can we change the id to something more descriptive, please? I.e. `mastery-stars`

File: templates/activity.mustache [hunk 3]
```diff
@@ -64,9 +64,40 @@
...
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd"/>
```
Comment (HAASLEWER) [inline templates/activity.mustache:70]: I haven't done it in Mustache but I think it does support macros, if it does can you move these definitions to a macro, please?

File: templates/activity.mustache [hunk 4]
```diff
@@ -64,9 +64,40 @@
...
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd" transform="translate(24)"/>
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd" transform="translate(48)"/>
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd" transform="translate(72)"/>
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd"  transform="translate(96)"/>
+                </symbol>
+            </defs>
+        </svg>
+        <div class="rating">
+        <!--   <div class="rating-bg" style="width: 90%;"></div> -->
```
Comment (HAASLEWER) [inline templates/activity.mustache:79]: Can this be removed?

File: templates/activity.mustache [hunk 5]
```diff
@@ -64,9 +64,40 @@
...
```
Comment (HAASLEWER) [inline templates/activity.mustache:97]: Can this be removed?

File: templates/activity.mustache [hunk 6]
```diff
@@ -64,9 +64,40 @@
...
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd" transform="translate(24)"/>
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd" transform="translate(48)"/>
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd" transform="translate(72)"/>
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd"  transform="translate(96)"/>
+                </symbol>
+            </defs>
+        </svg>
+        <div class="rating">
```
Comment (HAASLEWER) [inline templates/activity.mustache:96]: `rating` also doesn't make sense in this context.  Can we change it to something else, please?

File: templates/activity.mustache (hunk 1 above)
Comment (mareliefi) [reply templates/activity.mustache:67]: Hi @HAASLEWER yes it should be otherwise the format does not look right:)

**Pull request merged**